test/rantly_generator_test.rb

```diff
@@ -0,0 +1,46 @@
+require 'test_helper'
```

Comment:
I think that most of what is in rantly_test.rb is testing `rantly_generator`. You would have to move all related tests if you one to introduce a file for it.

Reply:
PR updated!